spec/spec_helper.rb

```diff
@@ -15,6 +15,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 require 'coveralls'
+require 'email_spec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.